### PR TITLE
Rollout improved example converter

### PR DESCRIPTION
**Pull request by t0yv0**
Rolls out improved TF example converter.

### CODE REVIEW

**Comment by VenelinMartinov** [comment]
Quite a few dropped examples here

**Comment by t0yv0** [comment]
Yes this was a case of a missing std reference; added now.

File: sdk/go/cloudflare/accessApplication.go [hunk 1]
```diff
@@ -22,53 +22,6 @@ import (
...
-// package main
-//
-// import (
-//
-//	"github.com/pulumi/pulumi-cloudflare/sdk/v5/go/cloudflare"
-//	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
-//
-// )
-//
-//	func main() {
```
Comment (iwahbe) [inline sdk/go/cloudflare/accessApplication.go:38]: We have lost quite a few examples. Do we know why?

**Review by iwahbe — APPROVED** [review]

**Comment by t0yv0** [comment]
```
t0yv0@Antons-MacBook-Pro> jq '.|select(.ExampleName=="#/resources/cloudflare:index/accessApplication:AccessApplication")' cov/byExample.json                                                                                                                 ~/bugs/ci-mgmt-889/pulumi-cloudflare
{
  "ExampleName": "#/resources/cloudflare:index/accessApplication:AccessApplication",
  "OriginalHCL": "resource \"cloudflare_access_application\" \"staging_app\" {\n  zone_id                   = \"0da42c8d2132a9ddaf714f9e7c920711\"\n  name                      = \"staging application\"\n  domain                    = \"staging.example.com\"\n  type                      = \"self_hosted\"\n  session_duration          = \"24h\"\n  auto_redirect_to_identity = false\n}\n\n# With CORS configuration\nresource \"cloudflare_access_application\" \"staging_app\" {\n  zone_id          = \"0da42c8d2132a9ddaf714f9e7c920711\"\n  name             = \"staging application\"\n  domain           = \"staging.example.com\"\n  type             = \"self_hosted\"\n  session_duration = \"24h\"\n  cors_headers {\n    allowed_methods   = [\"GET\", \"POST\", \"OPTIONS\"]\n    allowed_origins   = [\"https://example.com\"]\n    allow_credentials = true\n    max_age           = 10\n  }\n}\n",
  "ConversionResults": {
    "csharp": {
      "FailureSeverity": 2,
      "FailureInfo": "Duplicate resource \"cloudflare_access_application\" configuration: A cloudflare_access_application resource named \"staging_app\" was already declared at /e135.tf:1,1-55. Resource names must be unique per type in each module.",
      "Program": "",
      "TranslationCount": 1
    },
    "go": {
      "FailureSeverity": 2,
      "FailureInfo": "Duplicate resource \"cloudflare_access_application\" configuration: A cloudflare_access_application resource named \"staging_app\" was already declared at /e135.tf:1,1-55. Resource names must be unique per type in each module.",
      "Program": "",
      "TranslationCount": 1
    },
    "java": {
      "FailureSeverity": 2,
      "FailureInfo": "Duplicate resource \"cloudflare_access_application\" configuration: A cloudflare_access_application resource named \"staging_app\" was already declared at /e135.tf:1,1-55. Resource names must be unique per type in each module.",
      "Program": "",
      "TranslationCount": 1
    },
    "python": {
      "FailureSeverity": 2,
      "FailureInfo": "Duplicate resource \"cloudflare_access_application\" configuration: A cloudflare_access_application resource named \"staging_app\" was already declared at /e135.tf:1,1-55. Resource names must be unique per type in each module.",
      "Program": "",
      "TranslationCount": 1
    },
    "typescript": {
      "FailureSeverity": 2,
      "FailureInfo": "Duplicate resource \"cloudflare_access_application\" configuration: A cloudflare_access_application resource named \"staging_app\" was already declared at /e135.tf:1,1-55. Resource names must be unique per type in each module.",
      "Program": "",
      "TranslationCount": 1
    },
    "yaml": {
      "FailureSeverity": 2,
      "FailureInfo": "Duplicate resource \"cloudflare_access_application\" configuration: A cloudflare_access_application resource named \"staging_app\" was already declared at /e135.tf:1,1-55. Resource names must be unique per type in each module.",
      "Program": "",
      "TranslationCount": 1
    }
  }
}

```

With this:

```terraform
resource "cloudflare_access_application" "staging_app" {
  zone_id                   = "0da42c8d2132a9ddaf714f9e7c920711"
  name                      = "staging application"
  domain                    = "staging.example.com"
  type                      = "self_hosted"
  session_duration          = "24h"
  auto_redirect_to_identity = false
}

# With CORS configuration
resource "cloudflare_access_application" "staging_app" {
  zone_id          = "0da42c8d2132a9ddaf714f9e7c920711"
  name             = "staging application"
  domain           = "staging.example.com"
  type             = "self_hosted"
  session_duration = "24h"
  cors_headers {
    allowed_methods   = ["GET", "POST", "OPTIONS"]
    allowed_origins   = ["https://example.com"]
    allow_credentials = true
    max_age           = 10
  }
}

```

**Comment by t0yv0** [comment]
https://github.com/pulumi/pulumi-converter-terraform/issues/146

**Pull request merged**